debug/gdbserver.py

```diff
@@ -2217,8 +2217,10 @@ def main():
     selected_seed = parsed.seed
     if parsed.seed is None:
         selected_seed = int(datetime.now().timestamp())
-        print(f"PRNG seed for {target.name} is generated automatically")
-    print(f"PRNG seed for {target.name} is {selected_seed}")
+        if parsed.list_tests is None:
+            print(f"PRNG seed for {target.name} is generated automatically")
+    if parsed.list_tests is None:
+        print(f"PRNG seed for {target.name} is {selected_seed}")
```

Comment:
Looking at a greater detail - I would avoid these weird conditions here.

I would move seed generation and PRNG initialization code here:

```
1160     excluded_tests = load_excluded_tests(parsed.exclude_tests, target.name)                                                                               
1161     target.skip_tests += excluded_tests                                                                                                                   
1162                                                                                                                                                           
1163     results, count = run_tests(parsed, target, todo)
```

right before run_tests call.

Reply:
Thanks @aap-sc . Hopefully this addresses your feedback?

* https://github.com/riscv-software-src/riscv-tests/pull/561/commits/d3c0ecaf8629c889726a8deff1f00c4c1405101f

Reply:
yeah.. This is more to my liking. Thanks!

Reply:
I like it when there is lots of red color in MR :)